i18n/en.i18n.json

```diff
@@ -11,7 +11,7 @@
   "admin": "Admin",
   "post": "Post",
   "toolbox": "Toolbox",
-  "sign_up_sign_in": "Sign Up/Sign In",
+  "sign_up_sign_in": "Register/Sign In",
```

Comment:
we don't really have precedent for this as far as I'm aware, but would it be worth opening an issue to update the translations for this as well? It probably doesn't matter too much, but it seems like it would be good to be consistent